fix: Remove any dependencies to old doc preview - EXO-80993 (#1647)

mkrout opened the pull request:
This change will remove old documents preview and cleanup its dependencies, It will also and move search cards and ckeditor config from ecms

mkrout committed:
fix: Remove any dependencies to old doc preview - EXO-80993 (#1647)

This change will remove old documents preview and cleanup its dependencies

boubaker commented on documents-webapp/src/main/webapp/WEB-INF/conf/documents/ckeditor-configuration.xml:
```diff
@@ -0,0 +1,44 @@
+
+	This file is part of the Meeds project (https://meeds.io/).
+
+  Copyright (C) 2020 - 2024 Meeds Association contact@meeds.io
```
Please replace the License Header to AGPL

boubaker commented on documents-webapp/webpack.watch.js:
```diff
@@ -5,7 +5,7 @@ const webpackProductionConfig = require('./webpack.prod.js');
 module.exports = merge(webpackProductionConfig, {
   output: {
-    path: '/exo-server/webapps/documents-portlet/',
+    path: 'D:\\\eXo\\servers\\platform-7.1.x-meeds-qaui-SNAPSHOT/webapps/documents-portlet/',
```
```suggestion
    path: '/exo-server/webapps/documents-portlet/',
```

mkrout committed:
Update documents-webapp/webpack.watch.js

Co-authored-by: Boubaker Khanfir <bkhanfir@exoplatform.com>

mkrout committed:
fix: Update document preview for analytics - EXO-80992

This commit will use the new meeds documents preview for analytics instead of the old one.
